packages/@react-spectrum/dialog/docs/Dialog.mdx

```diff
@@ -43,52 +47,345 @@ import {Heading, Text} from '@react-spectrum/typography';
       <Header><Heading>The Heading</Heading></Header>
       <Divider />
       <Content>
-        <Text>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin sit amet tristique risus. In sit amet suscipit lorem. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. In condimentum imperdiet metus non condimentum. Duis eu velit et quam accumsan tempus at id velit. Duis elementum elementum purus, id tempus mauris posuere a. Nunc vestibulum sapien pellentesque lectus commodo ornare.</Text>
+        <Text>
+          This is a dialog.
+        </Text>
       </Content>
-      <Footer>
+      <ButtonGroup>
         <Button variant="secondary" onPress={close}>Cancel</Button>
         <Button variant="cta" onPress={close}>Confirm</Button>
-      </Footer>
+      </ButtonGroup>
     </Dialog>
   )}
 </DialogTrigger>
 ```
 
 ## Content
 
-*If the component has a children prop that accepts any type of content (e.g. `ReactNode`), include this section. Please include a note about how to internationalize the content.*
+A typical Dialog with a title, contents, and action buttons can be created like so:
+
+```tsx example
+<DialogTrigger>
+  <ActionButton>Publish</ActionButton>
+  {(close) => (
+    <Dialog>
+      <Header><Heading>Publish 3 pages</Heading></Header>
+      <Divider />
+      <Content>Confirm publish?</Content>
+      <ButtonGroup>
+        <Button variant="secondary" onPress={close}>Cancel</Button>
+        <Button variant="cta" onPress={close} autoFocus>Confirm</Button>
+      </ButtonGroup>
+    </Dialog>
+  )}
+</DialogTrigger>
+```
+
+A dismissable Dialog forgoes a footer section and its associated action buttons in favor
+of rendering a close button. If your Dialog is dismissable, make sure that you do not include a footer section or
+a ButtonGroup. While the Dialog will hide any elements occupying the footer section, it will not remove them from the DOM, 
+resulting in possible tab order issues.
```

Comment:
Is this true? `display: none` should hide the elements from the tab sequence as well I believe.

Reply:
ah lemme double check, I just remember that I had to add a `isDismissable` check to hide the footer buttons in the Dialog stories due to this issue, but it may have changed with the ButtonGroup update

Reply:
yeah, if you include a Footer with a checkbox or button, it messes up being able to tab to the dismissable Dialogs 'X' button since `getFocusableElementsInScope` in FocusScope picks up those hidden elements (maybe a bug in the selector matching?)

Also ButtonGroup doesn't get automatically hidden in dismissable Dialogs, is this bug as well? I can change the css to hide buttongroup if so

Reply:
Ah right, yeah. That's an issue with FocusScope.

Reply:
kk made https://jira.corp.adobe.com/browse/RSP-1640

Reply:
I have a possible fix in https://github.com/adobe-private/react-spectrum-v3/pull/401 to resolve the tab order stuff so I'll remove this bit if it gets merged in. I'll reword some of this stuff for now though